src/OmniSharp.Dnx/DnxPaths.cs

```diff
@@ -145,24 +145,30 @@ private IEnumerable<string> GetRuntimeLocations()
                 // Old path
                 yield return Path.Combine(home, ".kre");
             }
+
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ProgramFiles")))
+            {
+                yield return Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles"), "DNX");
+            }
+
+            yield return Path.Combine(@"/usr/local/lib/dnx");
         }
 
         private IEnumerable<string> GetRuntimePathsFromVersionOrAlias(string versionOrAlias, string runtimePath)
         {
             // Newer format
-            yield return GetRuntimePathFromVersionOrAlias(versionOrAlias, runtimePath, ".dnx", "dnx-mono.{0}", "dnx-clr-win-x86.{0}", "runtimes");
```

Comment:
It was not used in the current version, doesn't mean it wasn't used at one point in time.